# NDT 降采样相关
leaf_size: 0.4
# NDT 匹配相关
epsilon: 0.01   # 终止迭代的差值
resolution: 1   # 1
max_iteration: 35
step_size: 0.1    # 0.1

# IMU 噪声及零偏
imu_acc_noise: 1.8540218841840805e-01
imu_gyr_noise: 8.9867341184996230e-03
imu_acc_bias: 6.1931585615909864e-04
imu_gyr_bias: 2.1189007462883291e-05
imu_gravity: 9.8015

# 初始状态协方差矩阵和过程噪声协方差矩阵对角线值
# [ x,     y,      z,
#   roll,  pitch,  yaw,
#   vx,    vy,     vz,
#   vroll, vpitch, vyaw,
#   ax,    ay,     az    ]
initial_estimate_covariance:
  #  [1000.0, 1000.0, 1000.0, 1000.0, 1000.0, 1000.0, 1000.0, 1000.0, 1000.0, 1000.0, 1000.0, 1000.0, 0.001, 0.001, 0.001]
  [1000.0, 1000.0, 1000.0,
   1000.0, 1000.0, 1000.0,
   1000.0, 1000.0, 1000.0,
   1000.0, 1000.0, 1000.0,
   1000.0, 1000.0, 1000.0]
process_noise_covariance:
  [0.050, 0.050, 0.060,
   0.030, 0.030, 0.060,
   0.025, 0.001, 0.001,
   0.010, 0.010, 0.020,
   0.010, 0.001, 0.001]
